Source/WebCore/Modules/modern-media-controls/media/media-controller.js

```diff
@@ -221,6 +222,12 @@ class MediaController
             event.preventDefault();
         else if (event.type === this.fullscreenChangeEventType)
             this.host?.presentationModeChanged?.();
+        else if (event.type === "click" && event.target === this.media) {
```

Comment:
I think you'd do well to add a comment here explaining what you wrote in the commit message as it's not obvious to me that we're detecting a click _outside_ the video here:
> If the `<video>` receives a click event, and the `<video>` is also the target
of this event, this means we have clicked outside the border of the
media controls `<div>`, which covers the video content.

Reply:
Thanks! I just added this